packages/react-app/src/segments/Feedbox.tsx

```diff
@@ -149,12 +150,13 @@ function Feedbox() {
     if (loading || finishedFetching) return;
     setLoading(true);
     try {
-      const { count, results } = await api.fetchNotifications(
-        account,
-        NOTIFICATIONS_PER_PAGE,
+      const { count, results } = await EPNSAPI.fetchNotifications({
+        user: account,
+        pageSize: NOTIFICATIONS_PER_PAGE,
         page,
-        envConfig.apiUrl
-      );
+        chainId,
+        dev: true,
```

Comment:
@nilesh46 we are pointing here to `backend-dev` if you want to point to `backend-kovan` we can simply remove `dev`. please let us know which 1 is the preference to keep in the dApp code.

Reply:
It's already correct, dev dapp needs to be pointed to backend-dev.